pkg/operator/operator.go

```diff
@@ -1066,15 +1107,9 @@ func (o Operator) storageNotConfiguredMessage() string {
 
 	// if cluster version cannot be obtained due to any failure, point to the
 	// latest documentation
-	cv, err := o.client.GetClusterVersion(context.Background(), "version")
-	if err != nil {
-		klog.Warningf("failed to find the cluster version: %s", err)
-		return latestDocMsg
-	}
-
-	v, err := semver.Make(cv.Status.Desired.Version)
+	v, err := o.getClusterVersion()
```

Comment:
I would argue that CMO should pull the cluster version only once at startup and save it in the Operator struct but :shrug:

Reply:
I admit I just re-used the existing API supposing it works as expected (as it kind of already handles the same use case) but I agree we can do better. I reverted the reference linking as it'd be too much for this PR, also it was just a nice to have I wanted to add.
We could always add it back in another PR.